src/results/thermal_storage.jl

```diff
@@ -25,7 +25,7 @@ function add_hot_storage_results(m::JuMP.AbstractModel, p::REoptInputs, d::Dict,
     	soc = (m[Symbol("dvStoredEnergy"*_n)][b, ts] for ts in p.time_steps)
         r["soc_series_fraction"] = round.(value.(soc) ./ size_kwh, digits=3)
 
-        discharge = (m[Symbol("dvDischargeFromStorage"*_n)][b, ts] for ts in p.time_steps)
+        discharge = (sum(m[Symbol("dvHeatFromStorage"*_n)][b,q,ts] for b in p.s.storage.types.hot, q in p.heating_loads) for ts in p.time_steps)
```

Comment:
We could show the resulting "storage_to_load" for **each** quality, right?

Reply:
Yes! Storage to load has been added in 679c7b5e06a3511150fde6226717247fb8f7beb3

Reply:
Great, this will be good to know.